### ISSUE TITLE
Getting "You are not in a slate theme directory" after slate-migrate

### ISSUE THREAD

**Issue by thomasfaller** [Shopify/slate#537]
### Problem

I downloaded my theme and migrated it succesfully over to Slate (migrating assets into a scr/ folder) and then when I try to run a slate watch, it's giving me a "You are not in a slate theme directory" error.
Prior to that, migrating to Slate is automatically adding Slate-tools as dev-dependencies which seems to be causing even more problems.
I checked my path and directory.
Any thoughts?

### Replication steps

<img width="569" alt="screenshot 2018-05-08 15 35 19" src="https://user-images.githubusercontent.com/23104176/39763579-71647ebc-52d5-11e8-85a6-1844bec88902.png">

### More Information

ThemeKit 0.7.5 darwin/amd64
Slate 0.13.0
Mac OS 10.13.2

**Comment by thomasfaller** [comment]
Thanks @t-kelly I was able to fix this but I'm not too sure what's the fuss.
I have the 0.13.0 version of Slate installed globally. Whenever I work on a project requiring slate-tools, it's always the latest v1beta that's installed.

I opened my `node_modules/@shopify/slate-tools/package.json` and modified the version of `"slate-tools"` from `1.0.0-beta.1` to `0.13.0`, went through `npm install` again and everything works.

I guess the `slate` command was conflicting with the global v0 Slate I have (I'm waiting for the stable v1 to use it globally).

Or maybe I was doing something wrong here?
